experimental/traffic-portal/src/app/core/types/detail/type-detail.component.html

```diff
@@ -16,6 +16,8 @@
 	<tp-loading *ngIf="!type"></tp-loading>
 	<form ngNativeValidate (ngSubmit)="submit($event)" *ngIf="type">
 		<mat-card-header class="headers-container" *ngIf="!new">
+			<a mat-raised-button color="primary" *ngIf="type.useInTable === 'cachegroup'" [routerLink]="'/core/cache-groups'" [queryParams]="{typeName: type.name}">View Cache Groups</a>
+			<a mat-raised-button color="primary" *ngIf="type.useInTable === 'server'" [routerLink]="'/core/servers'" [queryParams]="{typeName: type.name}">View Servers</a>
```

Comment:
`typeName` was just an example. You have to use the actual property names from the API's response structure for each of table's objects. This doesn't work for servers, currently.

Reply:
in cache group type name is in  "typeName" and in servers type name is in "type". Made respective changes in redirect params.